workflows/docker-data-science/main.workflow

```diff
@@ -0,0 +1,13 @@
+workflow "run and plot pgbench performance" {
+  resolves = "generate figures"
+  on = "push"
+}
+
+action "generate data" {
+    uses = "./workflows/docker-data-science/docker"
```

Comment:
instead of creating an action, we can use [this python action](https://github.com/jefftriplett/python-actions) to install dependencies first, and then run this and the next action